HoloLensCommander/HoloLensCommander/Dialogs/GetAppInstallFilesDialogViewModelCommands.cs

```diff
@@ -96,6 +104,60 @@ public ICommand RemoveDependencyFileCommand
         }
 
         /// <summary>
+        /// Implementation of the browse for parents folder command.
+        /// </summary>
+        /// <returns>Task object used for tracking method completion.</returns>
+        private async Task BrowseForParentFolderAsync()
+        {
+            FolderPicker folderPicker = new FolderPicker();
+            folderPicker.ViewMode = PickerViewMode.List;
+            folderPicker.SuggestedStartLocation = PickerLocationId.ComputerFolder;
+            folderPicker.CommitButtonText = "Select";
+            folderPicker.FileTypeFilter.Add("*");
+
+            StorageFolder folder = await folderPicker.PickSingleFolderAsync();
+            if (folder!=null)
+            {
+                string appxFolderToken = StorageApplicationPermissions.FutureAccessList.Add(folder);
+                await AutoAddInstallFiles(appxFolderToken);
+            }
+        }
+
+        private async Task AutoAddInstallFiles(string appxFolderToken)
+        {
+            StorageFolder appPackageFolder = await StorageApplicationPermissions.FutureAccessList.GetFolderAsync(appxFolderToken);
+            IReadOnlyList<StorageFile> files = await appPackageFolder.GetFilesAsync();
+
+            Func<StorageFile, bool> appxFilter = f => f.FileType.Equals(".appx") || f.FileType.Equals(".appxbundle");
+            StorageFile appxFile = files.FirstOrDefault(appxFilter);
+            if (appxFile == null)
+            {
+                return;
+            }
+
+            this.AppPackageFile = appxFile;
+
+            StorageFile certFile = files.FirstOrDefault(f => f.FileType.Equals(".cer"));
+            if (certFile == null)
+            {
+                return;
+            }
+
+            this.AppCertificateFile = certFile;
+
+            DependencyFiles.Clear();
+            DependencyFileNames.Clear();
+
+            StorageFolder dependenciesFolder = await appPackageFolder.GetFolderAsync(@"Dependencies\x86");
```

Comment:
Radio buttons sound good.  The wrapper change I am planning will mostly dictate the architecture.  The notable exception would be for amd64/x64 architecture where x86 apps can also be installed.

I am going to investigate how straight forward it'd be to determine the architecture(s) contained in the appx/appxbundle